Sources/App/Scenes/CarPlaySceneDelegate.swift

```diff
@@ -0,0 +1,261 @@
+import CarPlay
+import Communicator
+import HAKit
+import PromiseKit
+import Shared
+
+
+public protocol EntitiesStateSubscription {
+    func subscribe()
+    func unsubscribe()
+}
+
+@available(iOS 16.0, *)
+class CarPlayDelegate: UIResponder {
+    public static let SUPPORTED_DOMAINS_WITH_STRING = [
```

Comment:
while android uses the capitalized string paradigm for constants, this is not common here.

this should be an enum like:

```swift
enum CarPlayDomain: CaseIterable {
    case button
    case cover
    case inputBoolean
    case input_button
    case light
    case lock
    case scene
    case script
    case switch

    var isMap: Bool { /* … */ }
    
    var domain: String {
        switch self {
            /* return value */
        }
    }
    
    var localizedDescription: String {
        switch self {
            /* return value */
        }
    }
}
```